src/components/sync-connected-sites.tsx

```diff
@@ -276,18 +295,27 @@ const SyncConnectedSitesSection = ( {
 										>
 											<div className="flex gap-2 pl-4 ml-auto shrink-0 h-5">
 												<Tooltip
-													text={ getLastSyncTimeWithType(
-														selectedSite.id,
-														connectedSite.id,
-														'pull'
-													) }
+													text={
+														isAnySiteSyncing
+															? isAnyConnectedSiteSyncing
+																? __(
+																		'This Studio site is syncing. Please wait for the sync to finish before you pull this site.'
```

Comment:
I wonder if it would make sense to make the following minor copy change - since "This Studio site" is already included in the beginning of the sentence:

```suggestion
																		'This Studio site is syncing. Please wait for the sync to finish before you pull it.'
```

Reply:
Adjusted in https://github.com/Automattic/studio/pull/718/commits/cd3a08cd9aaa7fe5ceb7ef4987467c76d85dd6b1